psi4/driver/procrouting/mcscf/mcscf_solver.py

```diff
@@ -44,6 +44,53 @@ def print_iteration(mtype, niter, energy, de, orb_rms, ci_rms, nci, norb, stype)
     core.print_out("%s %2d:  % 18.12f   % 1.4e  %1.2e  %1.2e  %3d  %3d  %s\n" %
                     (mtype, niter, energy, de, orb_rms, ci_rms, nci, norb, stype))
 
+def build_semicanonical_mos(ciwfn):
```

Comment:
To be `build` sounds like it should return those when its acting on the incoming Wavefunction. We may be consider calling this `semicanonical_orbs` and binding it to the CIWavefunction itself which should be very analogous to the `ci_nat_orb` function. What do you think?